src/CodeGenerator/src/Generator/CodeGenerator/PopulatorGenerator.php

```diff
@@ -273,4 +270,16 @@ private function generatePopulator(Operation $operation, StructureShape $shape,
             $classBuilder->addUse(Response::class);
         }
     }
+
+    private function generateListShapeMemberShape(Shape $memberShape, bool $forEndpoint): void
+    {
+        if ($memberShape instanceof StructureShape) {
+            $this->objectGenerator->generate($memberShape, $forEndpoint);
+        } elseif ($memberShape instanceof ListShape) {
+            $this->generateListShapeMemberShape($memberShape->getMember()->getShape(), $forEndpoint);
```

Comment:
If I understand correctly, we have a list inside a list.
I'm surprised we have nothing to do but call recursively `generateListShapeMemberShape`. I would expect the docbloc to be `foo[][]` instead of `foo[]` or things like that... 

Could you point me to the generated code or the needed operation?

Reply:
It is required by the `CalculateRouteMatrix` -> `RouteMatrix` response https://docs.aws.amazon.com/location/latest/APIReference/API_CalculateRouteMatrix.html#API_CalculateRouteMatrix_ResponseSyntax as otherwise the value objects are not generated.

The docblock is generated correctly https://github.com/async-aws/aws/pull/1472/files#diff-5b3b85f60b96fe85aa7ef355e6623dd98593119ca08bfa529d5349d54a68d346R42